lib/rspec/core/formatters/exception_formatter.rb

```diff
@@ -0,0 +1,136 @@
+module RSpec
+  module Core
+    module Formatters
+      # @private
+      class ExceptionFormatter
```

Comment:
I thought it was a good name, in spite of the fact that it's not a "Formatter" in the RSpec sense....but `ExceptionPrinter` is fine by me if you prefer it.